app/assets/javascripts/backbone/views/timeline_batch_actions_view.coffee

```diff
@@ -0,0 +1,27 @@
+class MS.Views.TimelineBatchActionsView extends Backbone.View
+
+  el: 'body'
+
+  events: (params) ->
+    "confirm:complete .batch-actions .batch-action": 'adjust_form'
+    "click #adjust-dates-confirm": 'hide_adjust_dates_modal'
+    "shown.bs.modal #adjust-dates": 'show_adjust_dates_modal'
+  
+  adjust_form: (e) ->
+    item = e.currentTarget
+    method_key = $(item).attr("data-method-key")
+    action_key = $(item).attr("data-action-key")
+
+    $form = $(item).closest("form")
+
+    $form.find("#set-method").attr("value", method_key)
```

Comment:
For things like this I recommend using e.g. `find('input[name=_method]')`. The input already has a unique attribute so it's better to select it using that instead of adding a new ID attrib to it. Try to minimize the number of attributes you're adding just so you can grab something from jQuery.